### DISCUSSION TITLE
PLANE DEVIATES FROM FP TILL I SNAP IT BACK ON COURSE

### DISCUSSION THREAD

**Discussion by polyefko** [flybywiresim/aircraft#2928] [Help] [answered]
I hope this is the correct area to post this for assistance. A32NX plane strays from the flight plan and slowly starts to turn let or right while all the PFD indicators are trying to direct the plane back to the flight path. For example if the plane slowly starts leaving the flight plan to the right the PFD indicator has moved to the left trying to get the plane to come back to the left to get back on line with flight path. But if I move my Joystick to the left forcing the plane ever so slightly to move back to the left it seems to snap the plane back to the course. Then 3 or 4 minutes later it starts all over again either left or right...I snap it back towards the course and the plane responds. I think I need to maybe set something in Navigation settings???? Thanks for your help.

**Comment by hotshotp** [comment, marked as answer]
known issue, custom autopilot is being worked on
